Apps/WebClient/src/ClientApp/src/components/timeline/entryCard/EntrycardTimelineComponent.vue

```diff
@@ -111,7 +111,10 @@ onMounted(() => {
                         </b-col>
                     </b-row>
                     <b-row>
-                        <b-col class="py-1 text-muted align-self-center">
+                        <b-col
+                            class="py-1 text-muted align-self-center"
+                            data-testid="entryCardDetailsSubtitle"
```

Comment:
There's some inconsistency between kebab-case and pascalCase.

see the data-testid in MedicationTimelineComponent

Reply:
Yeah, I prefer kebab case for data-testids, but we have more than a few that use camel case.  I settled on staying consistent with the others in this file.